scss/theme.scss

```diff
@@ -54,7 +67,7 @@
   font-family: 'Simplenote Tasks', sans-serif;
 
   .note-title {
-    font-size: 120%;
+    // font-size: 120%;
```

Comment:
is this commented-out on purpose? if so we should explain why in a comment

Reply:
Haha it's removed on purpose but I should remove the comment, yes 😂